rotkehlchen/db/addressbook.py

```diff
@@ -83,46 +104,54 @@ def add_addressbook_entries(
     def update_addressbook_entries(
             self,
             book_type: AddressbookType,
-            entries: list[AddressbookEntry],
+            entries: list[tuple[ChecksumEvmAddress, str, Optional[SupportedBlockchain]]],
     ) -> None:
         with self.write_ctx(book_type) as write_cursor:
```

Comment:
can you add a docstring comment explaining the behavior when the blockchain param is None and not?